packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -322,8 +300,28 @@ class SnackBar extends StatelessWidget {
       top: false,
       bottom: !isFloatingSnackBar,
       child: Row(
-        children: children,
         crossAxisAlignment: CrossAxisAlignment.center,
+        children: <Widget>[
+          SizedBox(width: snackBarPadding),
+          Expanded(
+            child: Container(
+              padding: const EdgeInsets.symmetric(vertical: _singleLineVerticalPadding),
+              child: DefaultTextStyle(
+                style: contentTextStyle,
+                child: content,
+              ),
+            ),
+          ),
+          if (action != null)
+            ButtonTheme(
+              textTheme: ButtonTextTheme.accent,
+              minWidth: 64.0,
+              padding: EdgeInsets.symmetric(horizontal: snackBarPadding),
+              child: action,
+            )
```

Comment:
Nit:
```suggestion
            ),
```

Reply:
We can't put a comma here because there's a `else` section